setup.py

```diff
@@ -442,7 +442,8 @@ def srcpath(name=None, suffix=".pyx", subdir="src"):
     "_libs.algos": {
         "pyxfile": "_libs/algos",
         "include": klib_include,
-        "depends": _pxi_dep["algos"],
+        "depends": _pxi_dep["algos"] + tseries_depends,
```

Comment:
I think you will need to add the khash_python file to the `depends` arguments of modules where it gets injected. Otherwise a change in `khash_python` might not re-trigger a build of libraries that use it

Reply:
just to make sure i understand: that should be a problem in the status quo too, not just in this PR, right?

Reply:
Hmm yea that's a good point. I think the the risk here is that there is a lot of code directly injected into the header, so I think you'd have to do a full clean / rebuild every time a change is made.

This is where setuptools is a real pain. Would be ideal if this could be implemented in a source file and just linked instead of directly injected via the header, but I don't know that setuptools has that kind of sophistication